enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/readreplica/EnterpriseReadReplicaEditionModule.java

```diff
@@ -266,6 +270,23 @@ private OnlineBackupKernelExtension pickBackupExtension( NeoStoreDataSource data
         dependencies.satisfyDependency( createSessionTracker() );
     }
 
+    static Predicate<String> fileWatcherFileNameFilter()
+    {
+        return Predicates.any(
+                fileName -> fileName.startsWith( PhysicalLogFile.DEFAULT_NAME ),
+                fileName -> fileName.startsWith( IndexConfigStore.INDEX_DB_FILE_NAME ),
+                filename -> filename.startsWith( StoreUtil.BRANCH_SUBDIRECTORY ),
+                filename -> filename.startsWith( StoreUtil.TEMP_COPY_DIRECTORY_NAME )
```

Comment:
Same as previous comment. I think this was just forgotten when the filters were inversed.